### PR TITLE
SS-2104 Docs home page

### PR DESCRIPTION
**Pull request by KaustubhKumar05**

### CODE REVIEW

File: components/Home/NewReleases.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                    mt: '$2',
+                    mb: '$8'
+                }}>
+                {date}
+            </Text>
+        </Box>
+    </Flex>
+);
+
+const NewReleases = () => (
```
Comment (hdz-666) [inline components/Home/NewReleases.tsx:73]: move the main comp to top

File: components/Home/PopularGuides.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+const PopularGuides = () => (
+    <Box css={{ w: '50%', '@xl': { w: '100%' } }}>
+        <Text variant="h6" css={{ color: '$textHighEmp', mt: '0' }}>
+            Popular how-to guides
+        </Text>
+        <Text variant="sm" css={{ color: '$textMedEmp', mt: '$2', mb: '$10' }}>
+            Explore breadth of capabilities with popular guides
+        </Text>
+        {guides.map((guide) => (
+            <Flex gap="2">
```
Comment (hdz-666) [inline components/Home/PopularGuides.tsx:15]: add pointer cursor

File: components/Home/NewReleases.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        link: '/react-native/v2/changelog/release-notes'
+    },
+    Server: {
+        icon: <ServerIcon style={iconStyle} />,
+        id: 'server',
+        link: '/server-side/v2/changelog/release-notes'
+    }
+};
+
+const ReleaseItem = ({ platform, version, date }) => (
```
Comment (hdz-666) [inline components/Home/NewReleases.tsx:48]: add pointer cursor

File: releases.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+const PLATFORM_WEB = 'Web';
+const PLATFORM_SERVER = 'Server';
+const PLATFORM_IOS = 'iOS';
+const PLATFORM_ANDROID = 'Android';
+const PLATFORM_REACT_NATIVE = 'React Native';
+const PLATFORM_FLUTTER = 'Flutter';
+
+
+export const releases = [
+    {
```
Comment (hdz-666) [inline releases.js:10]: why is this not driven via some sort of api?

**Review by hdz-666 — CHANGES_REQUESTED** [review]

File: releases.js (hunk 4 above)
Comment (KaustubhKumar05) [reply releases.js:10]: Will be updated by a script at build time in the next release

File: components/Home/NewReleases.tsx (hunk 1 above)
Comment (KaustubhKumar05) [reply components/Home/NewReleases.tsx:73]: Updated. Does this make a significant difference?

**Review by hdz-666 — APPROVED** [review]

**Pull request merged**